icu4c/source/tools/toolutil/writesrc.h

```diff
@@ -131,6 +134,30 @@ usrc_writeUCPTrieStruct(FILE *f,
 U_CAPI void U_EXPORT2
 usrc_writeUCPTrie(FILE *f, const char *name, const UCPTrie *pTrie, UTargetSyntax syntax);
 
+/**
+ * Writes the UnicodeSet serialized bytes and ranges list.
```

Comment:
"bytes" seems misleading and uninformative

Reply:
d528e355493ed019e493c9c9858280cb71613a55